engo_glfw.go

```diff
@@ -1,4 +1,4 @@
-//+build !netgo,!android
+// +build darwin,!arm,!arm64 linux windows
```

Comment:
> I'll take a look into it. I'm not sure you'd need a build tag for the emulator. 
`gomobile` makes binaries for emulators (amd64) whether you want or you don't. (Note that it looks like specifying architecture on `-target` like `-target ios:arm` of `gomobile` doesn't work for iOS. This seems TBD).

> Also why would I need to exclude android.
It's because android is one of linux machines and if you specify `linux` for build tags, this must include android machine unless you don't exclude android.